.golangci-new.yml

```diff
@@ -3,6 +3,7 @@
 
 run:
   timeout: 3m
+  allow-parallel-runners: true
```

Comment:
We don't use separate cache directories for separate runners, so using that option is not safe: golangci/golangci-lint#1769

That actually makes it clear that our complex way to run golangci-lint in parallel does not actually achieve that :) So let's simplify that by removing `deps` on other golangci-lint tasks